Package.swift

```diff
@@ -37,7 +38,8 @@ let package = Package(
 		.testTarget(
 			name: "PactSwiftTests",
 			dependencies: [
-				"PactSwift"
+				"PactSwift",
+        .product(name: "InlineSnapshotTesting", package: "swift-snapshot-testing")
```

Comment:
nitpick: Indentation to match "PactSwift". (applies to other spots too, but I'll nitpick only here 😉 )

reasoning: Because this is an open source project that could (hopefully 🤞) be picked up by other contributors in a very async manner it is important to stick to a certain code style (open to discussion on what that might be). It might not seem like a big deal now, but as the project evolves and contributions introduce code style deviance it gets harder and harder to read through the code and maintain it.